client/asset/eth/nodeclient_harness_test.go

```diff
@@ -143,6 +139,33 @@ func newRedeem(secret, secretHash [32]byte) *asset.Redemption {
 	}
 }
 
+// waitForReceipt waits for a tx. This is useful on testnet when a tx may be "missing"
+// due to reorg. Wait for a few blocks to find the main chain and hopefully our tx.
+func waitForReceipt(t *testing.T, nc *nodeClient, hash common.Hash) (*types.Receipt, error) {
+	t.Helper()
+	// Waiting as much as five blocks.
+	timesUp := time.After(time.Duration(5*secPerBlock) * time.Second)
+	for {
+		select {
+		case <-ctx.Done():
+			return nil, ctx.Err()
+		case <-time.After(time.Second):
+			receipt, err := nc.transactionReceipt(ctx, hash)
+			if err != nil {
+				if errors.Is(err, asset.CoinNotFoundError) {
+					fmt.Println("got um error", err.Error())
```

Comment:
just wondering, what's `um error` ?

Reply:
Lol. This wasn't meant to make it to the code. Will remove.